packages/components/src/font-size-picker/font-size-picker-select.tsx

```diff
@@ -59,7 +59,7 @@ const FontSizePickerSelect = ( props: FontSizePickerSelectProps ) => {
 				__experimentalHint: hint,
 			};
 		} ),
-		...( disableCustomFontSizes ? [] : [ CUSTOM_OPTION ] ),
+		...( pickerMode === 'both' ? [ CUSTOM_OPTION ] : [] ),
```

Comment:
For now, I opted to show the "custom" option only the picker mode is `both` — which means that, if the mode is `predefined` and there is a custom value, neither the toggle group nor the select dropdown have a way to visually show that.

I thought about showing the "custom" option in the select dropdown, but then I don't really know what do if the user selects it — normally, it would switch the picker type to `custom`, but if the picker mode is `predefined`, that is not an option.

Also, in case there are less than 5 predef font sizes, the UI would show a toggle group, where there is no visual indication that the user picked a custom value.

Reply:
For now I think it's ok to maintain the current behavior of what happens with `value={ /* some custom value */ }` and `disableCustomFontSizes={ true }`.

So, as observed on trunk, that would be:

- When predefs are a toggle group — Looks like no value is selected
- When predefs are a dropdown — "Custom" option

We'll just be clear in the docs that the consumer is responsible for any validation to prevent this state.

Reply:
That should the behavior already implemented in this PR. 

One difference I could spot, though. On trunk:
- Render the default example
- Switch to custom UI and input a custom value
- Using Storybook controls, set `disableCustomFontSizes` to `true`
- No UI renders!

In this PR:
- Render the default example
- Switch to custom UI and input a custom value
- Using Storybook controls, set `pickerMode` to `predefined`
- Un-selected toggle group control renders

I think this PR's behavior is better.

In any case, if we want to move forward with this approach, I can add more unit tests to "document" the expected behavior better.